hawkeye-maven-plugin/src/main/java/io/korandoru/hawkeye/maven/plugin/AbstractMojo.java

```diff
@@ -25,4 +25,7 @@ abstract class AbstractMojo extends org.apache.maven.plugin.AbstractMojo {
 
     @Parameter(name = "dryRun", defaultValue = "false")
     public boolean dryRun;
+
+    @Parameter(name = "skip", defaultValue = "false")
```

Comment:
You can add a properly named `hawkeye.skip`. This should be the idiom to skip from CLI.